src/js/providers/youtube.js

```diff
@@ -374,6 +374,7 @@ define([
 
             if (!item.image) {
```

Comment:
This `image` property could be added on setup so that we don't need new event handling:

https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/setup-steps.js#L218